### DISCUSSION TITLE
Feature request - icons on map instead of orange dots

### DISCUSSION THREAD

**Discussion by aliya-ryan** [digidem/mapeo-mobile#763] [Ideas]
Wondering if this could be in the long-term workplan - it would require very simple symbols for them to be visible at that scale, but I think lots of groups would like it as an option as it would mean the map they had looked more like a map they might be interacting with in MD and it would be easier to find and identify points from the map-view.

Particularly requested by Nīpodīmakī in Colombia

**Comment by arky** [comment]
I think this is a important feature request. Showing the icon it would help the mappers quickly understand the type of category at a glance. 

At the moment you can't differentiate the observations unless you select them preview the information.
